package.json

```diff
@@ -14,7 +14,7 @@
 	"license": "GPL-3.0",
 	"icon": "icon.png",
 	"engines": {
-		"vscode": "^1.44.0"
+		"vscode": "^1.82.0"
```

Comment:
Please lower the engine version to the minimal needed one - otherwise this plugin won't work with older versions.
For example 1.81 is the last one that _technically_ runs on Windows 7 (MS won't provide you with one, but VSCodium did); note: these don't necessarily need to match the devDependencies which can reference test packages that won't work with this old version.

... but: do we need a newer engine version for this change at all? If yes: why?